### ISSUE TITLE
Error building Appimage after latest update

### ISSUE THREAD

**Issue by namanlp** [linuxdeploy/linuxdeploy#272]
Hi there!

After the latest Arch Linux package updates, the LinuxDeploy is having errors, and can't build at all. The error is as shown below

```shell
ERROR: Strip call failed: /tmp/appimage_extracted_0299051fc9707c0ee1f44f4a07981aee/usr/bin/strip: tauri-example.AppDir/usr/lib/libffi.so.8: unknown type [0x13] section `.relr.dyn'

/tmp/appimage_extracted_0299051fc9707c0ee1f44f4a07981aee/usr/bin/strip: Unable to recognise the format of the input file `tauri-example.AppDir/usr/lib/libffi.so.8'
```

And same errors for a couple of libraries, like `libpango`, `libgstreamer` , `libgsttag` , `libgsttranscoder` etc.

## I have tried

- Installing a new system and updating that.
- Downgrading these packages fixes the issue.

Any insight on this issue?

**Comment by namanlp** [comment]
Also, using `NO_STRIP=true` environment variable compiles fine without errors

**Comment by hdclark** [comment]
Note this isn't strictly related to libffi.so.8, libpangoft2-1.0.so.0 has the same issue.

The current linuxdeply CI release succeeds on Debian stable, but fails when stripping on Arch Linux.

**Comment by namanlp** [comment]
Yeah, as I mentioned in the bug report itself, it is having issue with a bunch of libraries, so I guess it is issue with the LinuxDeploy itself.

Regarding Debian, it is not an issue right now, because the libraries are not updated for Debian yet. Downgrading only these libraries on Arch work fine as well. After a few weeks, the issue will be faced on Debian as well.

**Comment by vorporeal** [comment]
Should linuxdeploy be using the host system's `strip` instead of a bundled one?  If it's working with host system libraries, using a bundled `strip` (potentially from a different toolchain) is prone to issues like this.

**Comment by namanlp** [comment]
YES!

**Comment by FrancoRATOVOSON** [comment]
> Also, using `NO_STRIP=true` environment variable compiles fine without errors

Sorry, how do I use it ? Where do I put `NO_STRIP=true` ?

**Comment by hdclark** [comment]
@FrancoRATOVOSON Define it as an environment variable before running linuxdeploy. Here is an example where I unpack the linuxdeploy AppImage and then define NO_STRIP as a temporary environment variable for the invocation:  https://github.com/hdclark/DICOMautomaton/commit/35d7c4cc296cae41936b9945b10106a266bcbeb0

**Comment by TheAssassin** [comment]
GNU strip from binutils.

**Comment by lammmab** [comment]
Having this same issue.
```
Calling strip on library /home/lee/code/cpp/Courage-Reborn/pc/appimage-workspace/AppDir/usr/lib/libdxcompiler.so 
ERROR: Strip call failed: /tmp/.mount_linuxdkkFnfk/usr/bin/strip: /home/lee/code/cpp/Courage-Reborn/pc/appimage-workspace/AppDir/usr/lib/libdxcompiler.so: unknown type [0x13] section `.relr.dyn'
/tmp/.mount_linuxdkkFnfk/usr/bin/strip: Unable to recognise the format of the input file `/home/lee/code/cpp/Courage-Reborn/pc/appimage-workspace/AppDir/usr/lib/libdxcompiler.so'
```
Using the build from 1-alpha-20251107-1
